src/ff/galois_field.rs

```diff
@@ -19,7 +51,7 @@ fn assert_copy<C: Copy>(c: C) -> C {
 }
 
 macro_rules! bit_array_impl {
-    ( $modname:ident, $name:ident, $store:ty, $bits:expr, $arraylen:ty, $polynomial:expr ) => {
```

Comment:
Now `$bits` and `$arraylen` are defined in `Block` trait and passed to this macro as `$store`.